prow/config/cache.go

```diff
@@ -367,3 +370,80 @@ func (cache *InRepoConfigCache) get(
 	logrus.Error(err)
 	return nil, err
 }
+
+// InRepoConfigCacheStore wraps around an InRepoConfigCache, to allow it to be
```

Comment:
This intermediate hydration is useful because you don't want to reach out to the repos at startup?

Reply:
Yes --- reaching out to the repos at startup would be the only other way to warm up the cache to some degree. Saving the cache to disk and loading it back is better because it's probably much faster and simpler.